spec/2e_extensions-mechanism.adoc

```diff
@@ -49,8 +50,8 @@ See <<gpkg_extensions_sql>>.
 [caption=""]
 .Requirement 59
 ====
-Every extension of a GeoPackage SHALL be registered in a corresponding row in the gpkg_extensions table.
-The absence of a gpkg_extensions table or the absence of rows in the gpkg_extensions table SHALL both indicate the absence of extensions to a GeoPackage.
+In an Extended GeoPackage, every extension SHALL be registered in a corresponding row in the `gpkg_extensions` table. An extension SHALL NOT modify the definition or semantics of existing columns. An extension MAY define additional tables or columns. An extension MAY allow new values or encodings for existing columns.
```

Comment:
I'm not sure how else to describe this other than with an example. A tiles table has a tile_data blob. By default, it is JPG or PNG. The [WebP extension](http://www.geopackage.org/spec/#extension_tiles_webp) allows you to use WebP files instead. This doesn't change the semantics of the column, it just provides an additional option. If an app doesn't support WebP, it can't display the data for that layer but you can still use other data. The same applies to the elevation extension. Trying to render a DEM as if it were visual data is not going to give visually useful results, but it shouldn't break the app. 

Compare to the CRS WKT issue. If you change the semantics of the gpkg_spatial_ref_sys definition column from "old school" WKT to "newfangled" WKT and an application doesn't understand the new encoding, the whole GPKG is completely unusable.

If you can suggest another way to word this, I'm all ears.